tests/config.go

```diff
@@ -39,6 +46,15 @@ func NewConfig() *Config {
 	flag.StringVar(&cfg.TidbVersions, "tidb-versions", "v2.1.3,v2.1.4", "tidb versions")
 	flag.StringVar(&cfg.OperatorTag, "operator-tag", "master", "operator tag used to choose charts")
 	flag.StringVar(&cfg.OperatorImage, "operator-image", "pingcap/tidb-operator:latest", "operator image")
+	flag.StringVar(&cfg.GitRepoDir, "git-repo-dir", "/tidb-operator", "local directory to which tidb-operator cloned")
+	if home := homeDir(); home != "" {
+		flag.StringVar(&cfg.KubeConfigPath, "kubeconfig", filepath.Join(home, ".kube", "config"), "absolute path to the kubeconfig file")
+	} else {
+		// cannot get homedir, kube config file path must be provided explicitly
+		flag.StringVar(&cfg.KubeConfigPath, "kubeconfig", "", "absolute path to the kubeconfig file")
```

Comment:
I think use kubeconfig like [this](https://github.com/pingcap/private-tidb-operator/blob/master/cmd/volume-manager/main.go#L80) is simpler and clearer.  If you specify kubeconfig, it means running outside k8s cluster, otherwise it means running in k8s cluster.